tests/easier68k/assembler/basic_test_input.x68

```diff
@@ -0,0 +1,15 @@
+* Equates section
+start       EQU 400
+testData    EQU $ABCD
+
+* Code section
```

Comment:
we should definitely get a longer input for testing, but that can come later (maybe I can upload some previous HWs)

Reply:
Yup, that's why I named this the "basic" test input, it's used for testing a lot of the base functionality first.